Tasks/AzurePowerShell/AzurePowerShell.ps1

```diff
@@ -70,6 +70,11 @@ catch
    Write-Verbose "Unable to get the authScheme $error" 
 }
 
+if ($endpoint.Auth.parameters.authenticationType -ieq "spnCertificate")
```

Comment:
@vincentdass - Should we bring this file to master as V2 task instead of fixing in release branch.